### PR TITLE
SecurityInsights 0.2.0 release

### PR DESCRIPTION
**Pull request by dicolanl**
## Description

GA Release of module.
Updated help files
minor updates based on API changes.

## Checklist

- [X] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [X] The title of the PR is clear and informative
- [X] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [X] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [X] If applicable, the changes made in the PR have proper test coverage
- [X] For public API changes to cmdlets:
    - [X] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
        - NET SDK Update: https://github.com/Azure/azure-sdk-for-net/pull/19359
        - Approved API Breaking change: https://github.com/Azure/azure-rest-api-specs/pull/13137
        - Approved API Breaking change: https://github.com/Azure/azure-rest-api-specs/pull/12556
    - [X] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/SecurityInsights/SecurityInsights/ChangeLog.md [hunk 1]
```diff
@@ -20,5 +20,8 @@
 
 ## Upcoming Release
 
+## Version 1.0.0
+* GA Release
+
```
Comment (dcaro) [inline src/SecurityInsights/SecurityInsights/ChangeLog.md:25]: As discuss, let's not make this GA for the time being.

File: src/SecurityInsights/SecurityInsights/Az.SecurityInsights.psd1 [hunk 2]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.1.0'
+ModuleVersion = '1.0.0'
```
Comment (dcaro) [inline src/SecurityInsights/SecurityInsights/Az.SecurityInsights.psd1:15]: Let's make this 0.2.0?

File: src/SecurityInsights/SecurityInsights/Az.SecurityInsights.psd1 [hunk 3]
```diff
@@ -12,7 +12,7 @@
 # RootModule = ''
 
 # Version number of this module.
-ModuleVersion = '0.1.0'
+ModuleVersion = '0.2.0'
```
Comment (VeryEarly) [inline src/SecurityInsights/SecurityInsights/Az.SecurityInsights.psd1:15]: please make this version remain 0.1.0, Azure-PowerShell team will handle the version when release

File: src/SecurityInsights/SecurityInsights/Az.SecurityInsights.psd1 [hunk 4]
```diff
@@ -133,7 +133,7 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '* Initial Release'
+        ReleaseNotes = 'Added Teams to Office Data Connector, Bug Fixes and updated documentation.'
```
Comment (VeryEarly) [inline src/SecurityInsights/SecurityInsights/Az.SecurityInsights.psd1:136]: same here, please revert it back to previous release notes

File: src/SecurityInsights/SecurityInsights/ChangeLog.md [hunk 5]
```diff
@@ -20,5 +20,10 @@
 
 ## Upcoming Release
 
+## Version 0.2.0
```
Comment (VeryEarly) [inline src/SecurityInsights/SecurityInsights/ChangeLog.md:23]: please remove line 22-23

**Review by VeryEarly — CHANGES_REQUESTED** [review]
please also update PR title because SecurityInsights is not going to be GAed and this title will became commit name which could be misleading.

File: src/SecurityInsights/SecurityInsights/Az.SecurityInsights.psd1 (hunk 4 above)
Comment (dicolanl) [reply src/SecurityInsights/SecurityInsights/Az.SecurityInsights.psd1]: fixed

File: src/SecurityInsights/SecurityInsights/ChangeLog.md (hunk 5 above)
Comment (dicolanl) [reply src/SecurityInsights/SecurityInsights/ChangeLog.md]: fixed.

**Comment by dicolanl** [comment]
@VeryEarly made the changes requested including PR titile.

**Review by VeryEarly — APPROVED** [review]

**Pull request merged**